From 28967771ce0e46c52592b694a6c5e12997d13966 Mon Sep 17 00:00:00 2001
From: Alistair Francis <alistair@alistair23.me>
Date: Fri, 11 Oct 2024 18:11:17 +1000
Subject: [PATCH] daemon: Fixup SMS sending

Commit bf7951bfd16a "Initialize KCrash" broke SMS sending with Spacebar
as the name of the Daemon doesn't match what is expected.

Let's revert the name change to keep things consistent.

Fixes: bf7951bfd16a "Initialize KCrash"
Resolves: https://bugs.kde.org/show_bug.cgi?id=494381
Signed-off-by: Alistair Francis <alistair@alistair23.me>
---
 daemon/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/main.cpp b/daemon/main.cpp
index ce36c0f..35d0cf2 100644
--- a/daemon/main.cpp
+++ b/daemon/main.cpp
@@ -22,7 +22,7 @@ int main(int argc, char *argv[])
 
     QCoreApplication app(argc, argv);
 
-    KAboutData about(u"spacebar-daemon"_s, u"Spacebar"_s, QStringLiteral(SPACEBAR_VERSION_STRING));
+    KAboutData about(u"Spacebar"_s, u"Spacebar"_s, QStringLiteral(SPACEBAR_VERSION_STRING));
     KAboutData::setApplicationData(about);
 
     KCrash::initialize();
-- 
GitLab

